.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
I'm not sure this file is necessary.

Reply:
Yes, I think we can add it to .gitignore.

Reply:
I added the file to .gitignore but now I can't see it